src/main/GameReader.ts

```diff
@@ -95,13 +95,13 @@ export default class GameReader {
 					break;
 			}
 
-			let allPlayersPtr = this.readMemory<number>('ptr', this.gameAssembly.modBaseAddr, this.offsets.allPlayersPtr) & 0xffffffff;
```

Comment:
What's the difference between PTR and POINTER?

They were different in the original, and you have gone as far as including them in the enum, yet you have converted all instances of `ptr` to `DataType.POINTER`.

If they are exactly the same, why have both in the enum?
If they are different, then don't change existing usages of `ptr` to `DataType.POINTER`